azure-pipelines.yml

```diff
@@ -41,38 +42,43 @@ stages:
           arguments: '/output buildserver /nofetch'
 
       - task: DotNetCoreCLI@2
+        displayName: dotnet build
         inputs:
           command: 'build'
           projects: 'wwt-website-net5.slnf'
           arguments: '--configuration $(buildConfiguration)'
-        
+
       - task: DotNetCoreCLI@2
+        displayName: dotnet test
         inputs:
           command: 'test'
           projects: 'wwt-website-net5.slnf'
           arguments: '--configuration $(buildConfiguration)'
 
       - task: DotNetCoreCLI@2
+        displayName: dotnet publish
         inputs:
           command: 'publish'
           publishWebProjects: false
           zipAfterPublish: true
           modifyOutputPath: true
           projects: 'src/WWT.Web/WWT.Web.csproj'
-          arguments: '--configuration $(buildConfiguration) -o $(Build.ArtifactStagingDirectory)/website'
-      
+          arguments: '-f net5.0 -c $(buildConfiguration) -o $(Build.ArtifactStagingDirectory)/website'
```

Comment:
this shouldn't be necessary

Reply:
I ran into an issue in this area when working on setting up a Linux Docker build to add to the pipeline. But I agree that I now think that this particular change is not needed. (I tried changing `<TargetFramework>` to `<TargetFrameworks>` and that caused issues.)

Reply:
Specifically, without this PR:

```
$ dotnet publish -c Release -o out wwt-website-net5.slnf 
Microsoft (R) Build Engine version 16.8.0+126527ff1 for .NET
Copyright (C) Microsoft Corporation. All rights reserved.

  Determining projects to restore...
  All projects are up-to-date for restore.
/usr/share/dotnet/sdk/5.0.100/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.CrossTargeting.targets(27,5): error NETSDK1129: The 'Publish' target is not supported without specifying a target framework. The current project targets multiple frameworks, you must specify the framework for the published application. [/a/wwt/sites/wwt-website/tests/WWT.Azure.Tests/WWT.Azure.Tests.csproj]
/usr/share/dotnet/sdk/5.0.100/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.CrossTargeting.targets(27,5): error NETSDK1129: The 'Publish' target is not supported without specifying a target framework. The current project targets multiple frameworks, you must specify the framework for the published application. [/a/wwt/sites/wwt-website/tests/WWT.PlateFiles.Tests/WWT.PlateFiles.Tests.csproj]
/usr/share/dotnet/sdk/5.0.100/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.CrossTargeting.targets(27,5): error NETSDK1129: The 'Publish' target is not supported without specifying a target framework. The current project targets multiple frameworks, you must specify the framework for the published application. [/a/wwt/sites/wwt-website/src/WWT.PlateFiles/WWT.PlateFiles.csproj]
/usr/share/dotnet/sdk/5.0.100/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.CrossTargeting.targets(27,5): error NETSDK1129: The 'Publish' target is not supported without specifying a target framework. The current project targets multiple frameworks, you must specify the framework for the published application. [/a/wwt/sites/wwt-website/tests/WWT.Caching.Tests/WWT.Caching.Tests.csproj]
/usr/share/dotnet/sdk/5.0.100/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.CrossTargeting.targets(27,5): error NETSDK1129: The 'Publish' target is not supported without specifying a target framework. The current project targets multiple frameworks, you must specify the framework for the published application. [/a/wwt/sites/wwt-website/tests/WWT.Providers.Tests/WWT.Providers.Tests.csproj]
```

I don't know why the publish command in the Pipelines Linux build doesn't have these problems, but the above is what I currently understand to be needed to make a Docker image.

Reply:
Aha, I can get away with more minimal changes if I flip some `<TargetFrameworks>` to the singular.